Classes/Engine/HandlebarsEngine.php

```diff
@@ -167,7 +167,18 @@ protected function getOptions() : array
                 },
                 'lookup' => function ($labels, $key) {
                     return isset($labels[$key]) ? $labels[$key] : '';
-                }
+                },
+                'ifEqual' => function () {
+                    $arguments = func_get_args();
+                    $context = array_pop($arguments);
+                    return count(array_unique($arguments)) === 1 ? $context['fn']() : ($context['inverse'] ? $context['inverse']() : '');
```

Comment:
Could you please explain this `? $context['fn']() : ($context['inverse'] ? $context['inverse']() : ''` pattern? What is it used for?

Reply:
The fn is the function like the if part and the inverse the else. In the case of the if it could be that you don't have an else this is why I need to check if there is a inverse part.